inst/templates/Makefile

```diff
@@ -0,0 +1,29 @@
+R = R --vanilla CMD
```

Comment:
I don't really think this is necessary, I would drop this and just use `R CMD` directly in the build steps.

Also are we trying to make this work on Windows? or not?

Reply:
I rarely write anything on Windows these days, personally, and I've never written `make` recipes that don't specifically rely on `sh` or `bash`. If someone has a quick `ifeq()` block with support, I'm all ears, but I'm fine stuffing Windows-specific support for this particular PR into the backlog.

Reply:
And regarding your first comment: not passing `--vanilla` will fail if it tries to run and some tools aren't installed, right? Like `pandoc` for building the manual, etc.? Or did you just mean don't add a Makevar for the whole command `R --vanilla CMD`?

Reply:
Edit: I've ran some ad-hoc stuff, I might have gotten ahead of myself. Updated.